### ISSUE TITLE
When will you release the codes to run on KAIST?

### ISSUE THREAD

**Issue by shelkspa** [Jamie725/RGBT-detection#3]
Hi, I am very interested in your work. I ran the current code on KAIST but I can't get such LAMR mentioned in your paper. When will you release the codes to run on KAIST? Could you share the trained model on KAIST?

**Comment by Jamie725** [comment]
Yes I'll release the codes on KAIST as well in the short future, please stay tuned.
